scripts/centos_jswarm.sh

```diff
@@ -30,5 +30,6 @@ ansible-playbook -i "${INVENTORY}" \
 
 echo "Committing container at tag ${CINCH_VERSION}"
 docker commit \
-	--change 'ENTRYPOINT ["/usr/local/bin/dockerize", "-template", "/etc/sysconfig/jenkins_swarm:/etc/sysconfig/jenkins_swarm.templated", "/opt/jswarm.sh"]' \
+	--change 'USER jenkins' \
+	--change 'ENTRYPOINT ["/usr/local/bin/dockerize", "-template", "/etc/sysconfig/jenkins_swarm:/var/lib/jenkins/jenkins_swarm.templated", "/opt/jswarm.sh"]' \
```

Comment:
Are you referring to the /opt/jswarm.sh or is this paralleling the comment about the config file below?